docs/eventing/samples/cloudauditlogssource/README.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Global: for file names, we should be separating words with hyphens. For example, this should be /samples/cloud-audit-logs-source/ instead.